vnext/include/ReactUWP/Views/ShadowNodeBase.h

```diff
@@ -74,6 +74,7 @@ struct REACTWINDOWS_EXPORT ShadowNodeBase : public facebook::react::ShadowNode {
   virtual void AddView(ShadowNode &child, int64_t index) override;
   virtual void RemoveChildAt(int64_t indexToRemove) override;
   virtual void createView() override;
+  virtual bool needsForceLayout();
```

Comment:
>needsForceLayout(); [](start = 15, length = 19)

Nit: Camel case for first letter. 
NeedsForceLayout() const
or
IsForceLayout() const